tools/runners/energy_factory_runner.py

```diff
@@ -88,4 +93,103 @@ def verify_energy_factory(args: Any):
     verify_contracts(args, [energy_factory_address])
     
     print("All contracts have been verified.")
-    
\ No newline at end of file
+    
+
+def generate_unlock_tokens_transactions(args: Any):
+    """Generate unlock tokens transactions"""
+
+    context = Context()
+
+    exported_accounts_path = args.accounts_export
+    function_name = "unlockEarly"
+
+    if not exported_accounts_path:
+        print("Missing required arguments!")
+        return
+
+    network_providers = NetworkProviders(config.DEFAULT_API, config.DEFAULT_PROXY)
+    network_providers.network = network_providers.proxy.get_network_config()
+    chain_id = network_providers.proxy.get_network_config().chain_id
+    config_tx = DefaultTransactionBuildersConfiguration(chain_id=chain_id)
+    signature = get_default_signature()
+    default_account = Account(None, config.DEFAULT_OWNER)
+    default_account.sync_nonce(network_providers.proxy)
+
+    exported_accounts = read_accounts_from_json(exported_accounts_path)
+
+    fund_shadowfork_accounts(exported_accounts)
+    sleep(30)
+
+    with ThreadPoolExecutor(max_workers=500) as executor:
+        exported_accounts = list(executor.map(sync_account_nonce, exported_accounts))
+
+    energy_contract: SimpleLockEnergyContract = context.get_contracts(config.SIMPLE_LOCKS_ENERGY)[0]
+    accounts_with_token = get_acounts_with_token(exported_accounts, energy_contract.locked_token)
+
+    print(f"Found {len(accounts_with_token)} accounts with token {energy_contract.locked_token}")
+
+    transactions = []
+    accounts_index = 1
+    for account_with_token in accounts_with_token:
+        print(f"Processing account {accounts_index} / {len(accounts_with_token)}")
+
+        account = Account(pem_file=config.DEFAULT_OWNER)
+        account.address = WrapperAddress.from_bech32(account_with_token.address)
+        account.nonce = account_with_token.nonce
+
+        tokens = [
```

Comment:
wonders of python :laughing:

Reply:
true. list comprehension looks like voodoo magic sometime